src/rocker/extensions.py

```diff
@@ -238,6 +239,10 @@ def register_arguments(parser, defaults={}):
             action='store_true',
             default=defaults.get('user-preserve-home', False),
             help="Do not delete home directory if it exists when making a new user.")
+        parser.add_argument('--use-user-shell',
```

Comment:
I proposed `--user-override-shell` 

The beginning of the argument matches the plugin name which I definitely want. All arguments relating to that plugin should start with `--user` for grouping and visibility.